### ISSUE TITLE
Netlify and GH actions issues

### ISSUE THREAD

**Issue by NukeManDan** [substrate-developer-hub/substrate-docs#1160]
#1159 shows that the workflows on github are now broken - @jonnysmillie are they needed and/or need modification? and also the netlify integration with branches may be messed up now, with name changes etc. I think a full review of the CI would be in order :+1:

**Comment by jonnysmillie** [comment]
Agreed. I would like to remove the current incarnation of link checking. My suggestion would be to remove the link checker as part of the deployment process and add some link checker as part of the local build process or that can somehow be used outside of the deployment.

Suggestions:
https://www.gatsbyjs.com/plugins/gatsby-remark-check-links/
https://silvenon.com/blog/checking-for-dead-urls
https://matthewsetter.com/writing-tools/npm-broken-link-checker/

**Comment by jonnysmillie** [comment]
I have removed the link checker from the Github actions on this repo for now.
